src/makeCalculator.js

```diff
@@ -4,7 +4,69 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  function add(a) {
+    return (result += a);
+  }
+
+  function subtract(a) {
+    return (result -= a);
+  }
+
+  function multiply(a) {
+    return (result *= a);
+  }
+
+  function divide(a) {
+    if (a === 0) {
+      return;
+    }
+
+    return (result /= a);
+  }
+
+  return {
+    get result() {
+      return result;
+    },
+
+    operate(method, num) {
+      method(num);
```

Comment:
In the `operate` method, you should check if `method` is a function before calling it. You can do this by using `typeof method === 'function'`. This ensures that the method passed is callable, which aligns with the task requirements.